client/lib/menu-data/test/test-menu-data.js

```diff
@@ -297,29 +297,6 @@ describe( 'MenuData', function() {
 		} );
 	} );
 
-	describe( 'saveMenu', function () {
```

Comment:
Unfortunately, this whole code was one of the first parts of Calypso.

Well, it is not matter of updating, these tests are 'pretend'.
These tests right now test random things that have nothing to do with the whole shape of this code.
Furthermore, I strongly believe the component code has to be rewritten.

I agree we need to provide these tests, but it is definitely job for a different PR and question arises if we have now the time to do it.
I will try to estimate the work needed to write proper tests here.

Problem is: the MenuData has the weird shape it has now (with prototype methods for random stuff) just to make it easier to stub and the code is more convoluted.

Thats why I opted to rip out useless test, because if i start rewriting it to be proper, I have no idea where will i finish.
I can make it pass, this is not a problem, but it will still be "pretend" as now - it makes us only feel better that we have tests, the value of it is minimal.